core/src/main/java/org/hisp/dhis/android/core/category/CategoryCategoryOptionLinkStoreImpl.java

```diff
@@ -54,9 +133,39 @@ private int executeInsert() {
         return lastId;
     }
 
-    @Override
-    public int delete() {
-        return databaseAdapter.delete(CategoryCategoryOptionLinkModel.TABLE);
+    private boolean wasExecuted(int numberOfRows) {
+        return numberOfRows >= 1;
+    }
+
+    private boolean execute(SQLiteStatement statement) {
+        int rowsAffected = databaseAdapter.executeUpdateDelete(CategoryComboModel.TABLE, statement);
+        statement.clearBindings();
+
+        return wasExecuted(rowsAffected);
+    }
+
+    private List<CategoryCategoryOptionLinkModel> mapCategoryCategoryOptionLinkModelsFromCursor(
```

Comment:
this name is large, How only there is one method to map from cursor a name as mapFromCursor is more readable, can you rename it?

Reply:
Sure!